internal/connect/hwinfo.go

```diff
@@ -31,6 +32,7 @@ type hwinfo struct {
 	Arch          string `json:"arch"`
 	UUID          string `json:"uuid"`
 	CloudProvider string `json:"cloud_provider"`
+	MemTotal      int    `json:"memtotal,omitempty"`
```

Comment:
`mem_total` reads easier and would be consistent with the previous line.

Reply:
Sure thing!

Reply:
Done